example/trivial/datadog/datadog-config.json

```diff
@@ -0,0 +1,6 @@
+{
+  "service": "nginx",
+  "operation_name_override": "nginx.handle",
```

Comment:
Are these the values we want users to use in normal setups?
This part of the example will be copied verbatim by users, so make sure you like these values! 😄